sdk/core/System.ServiceModel.Rest/src/PipelineMessage.cs

```diff
@@ -48,7 +48,12 @@ public virtual PipelineResponse Response
     public void SetProperty(Type type, object value) =>
         _propertyBag.Set((ulong)type.TypeHandle.Value, value);
 
-    public virtual ResponseErrorClassifier ResponseClassifier { get; set; } = InvocationOptions.DefaultResponseClassifier;
+    private ResponseErrorClassifier _responseClassifier = RequestOptions.DefaultResponseClassifier;
```

Comment:
This is the issue - on Azure.Core's HttpMessage I would like to change the type of the property, but this is not allowed per D.11.7.

**The .NET BCL team considers this an unacceptable breaking change, and would recommend adding a new method or property if this functional change is required.**

Reply:
What if we did a "delegate" decomposition - have an "ErrorClassifier" and a "RetryClassifier" and Azure.Core "ResponseClassifier" can compose the two?